CRM/Batch/Form/Entry.php

```diff
@@ -1172,14 +1163,6 @@ protected function legacyProcessMembership($contactID, $membershipTypeID, $custo
     //since we are renewing,
     //make status override false.
     $memParams['is_override'] = FALSE;
-
-    //CRM-4027, create log w/ individual contact.
-    if ($modifiedID) {
-      // @todo this param is likely unused now.
-      $memParams['is_for_organization'] = TRUE;
-    }
-    $params['modified_id'] = $modifiedID ?? $contactID;
```

Comment:
``$params`` is a misnamed variable which is never used - hence setting this does nothing. Also - the Membership BAO now makes some assumptions about using the logged in user so there is no case to be made for fixing rather than removing this line